northstar/src/runtime/island/mod.rs

```diff
@@ -64,20 +65,23 @@ type Container = MountedContainer<IslandProcess>;
 pub(super) struct Island {
     tx: EventTx,
     config: Config,
+    tripwire_read: PipeRead, // Used by child process to detect if the parent process has died
+    tripwire_write: PipeWrite,
 }
 
 pub(super) enum IslandProcess {
     Created {
         pid: Pid,
         exit_status: Box<dyn Future<Output = Result<ExitStatus, Error>> + Unpin + Send + Sync>,
         io: (Option<io::Log>, Option<io::Log>),
-        _dev: Dev,
         checkpoint: Checkpoint,
+        _dev: Dev,
     },
     Started {
         pid: Pid,
         exit_status: Box<dyn Future<Output = Result<ExitStatus, Error>> + Unpin + Send + Sync>,
         io: (Option<io::Log>, Option<io::Log>),
+        checkpoint: Checkpoint,
```

Comment:
Think the `Checkpoint` isn't needed one the process is started. This can be removed here.